### PR TITLE
Upgrade to v19

### PR DESCRIPTION
**Pull request by huffmanca**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Upgrades dependencies to v0.19.0-rc2, so that a tag may be created from this new version.

**Does this PR introduce a user-facing change?**:
```release-note
Updates kubernetes dependencies to v0.19.0-rc2.
```

### CODE REVIEW

**Comment by xing-yang** [comment]
I have asked @Kartik494 to update to 1.19 as part of https://github.com/kubernetes-csi/external-snapshotter/pull/307, but I'm fine to have this done separately.

**Comment by huffmanca** [comment]
@xing-yang From discussions with Jan this morning, we're hoping to have a new release cut with the 1.19 dependencies very soon. I'd prefer to get these changes in sooner if at all possible.

File: pkg/snapshotter/snapshotter.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ func (s *snapshot) CreateSnapshot(ctx context.Context, snapshotName string, volu
 		return "", "", time.Time{}, 0, false, err
 	}
 
-	klog.V(5).Infof("CSI CreateSnapshot: %s driver name [%s] snapshot ID [%s] time stamp [%d] size [%d] readyToUse [%v]", snapshotName, driverName, rsp.Snapshot.SnapshotId, rsp.Snapshot.CreationTime, rsp.Snapshot.SizeBytes, rsp.Snapshot.ReadyToUse)
+	klog.V(5).Infof("CSI CreateSnapshot: %s driver name [%s] snapshot ID [%s] time stamp [%v] size [%d] readyToUse [%v]", snapshotName, driverName, rsp.Snapshot.SnapshotId, rsp.Snapshot.CreationTime, rsp.Snapshot.SizeBytes, rsp.Snapshot.ReadyToUse)
```
Comment (xing-yang) [inline pkg/snapshotter/snapshotter.go:76]: Can you remove this change?  This is not related to upgrading to v19.

Comment (huffmanca) [reply pkg/snapshotter/snapshotter.go]: I've removed the change.

It was included because unit tests were failing locally without making this modification.

Comment (xing-yang) [reply pkg/snapshotter/snapshotter.go]: Can you submit a separate PR for this?

Comment (huffmanca) [reply pkg/snapshotter/snapshotter.go]: This has been submitted as https://github.com/kubernetes-csi/external-snapshotter/pull/338

**Comment by xing-yang** [comment]
Do you run `go mod tidy` and `go mod vendor`?

**Comment by huffmanca** [comment]
> Do you run `go mod tidy` and `go mod vendor`?

I did. This should have happened in the second commit.

File: go.mod [hunk 2]
```diff
@@ -5,61 +5,42 @@ go 1.12
...
-	k8s.io/api v0.18.0
-	k8s.io/apimachinery v0.18.0
-	k8s.io/client-go v0.18.0
-	k8s.io/code-generator v0.18.0
-	k8s.io/component-base v0.18.0
+	k8s.io/api v0.19.0-rc.2
+	k8s.io/apimachinery v0.19.0-rc.2
+	k8s.io/client-go v0.19.0-rc.2
+	k8s.io/code-generator v0.19.0-rc.2
+	k8s.io/component-base v0.19.0-rc.2
```
Comment (xing-yang) [inline go.mod:20]: I'll merge this.

Can you submit another PR to move klog from v1 to v2?  Like this one: https://github.com/kubernetes-csi/external-attacher/pull/238/files

Comment (huffmanca) [reply go.mod:20]: This has been submitted at https://github.com/kubernetes-csi/external-snapshotter/pull/340 .
